src/pages/User/LoginRegister.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { Box, Flex, Button, Image } from '@chakra-ui/react';
+import LoginInput from '../../components/LoginRegister/LoginInput';
+import RegisterInput from '../../components/LoginRegister/RegisterInput';
+import logo from '../../images/logo-black.png';
+
+function LoginRegister() {
+  const [loginRegister, setLoginRegister] = useState('login');
+
+  return (
+    <Flex minHeight="100vh" h="100%" w="100vw" justify="center" bg="background.login">
+      <Box minW="35%" marginBottom="50px">
+        <Flex justify="center" marginTop="36px" marginBottom="36px">
+          <Image src={logo} w="350px"/>
+        </Flex>
+        <Box borderRadius="lg" borderWidth="1px" bg="white">
+          <Flex direction="row" justify="space-between" margin="20px 100px 24px 100px">
+            <Box>
+              <Button
+                opacity={loginRegister === 'login' ? '1' : '0.5'}
+                variant="unstyled"
+                _focus={{
+                  outline: 'none',
+                }}
+                _hover={{
+                  color: 'brand.50',
+                }}
+                onClick={() => setLoginRegister('login')}
+              >
+                Sign In
+              </Button>
+              {loginRegister === 'login' && <Box bg="brand.50" h="5px" borderRadius="sm"></Box>}
```

Comment:
this is a nit but best code quality would probably be to abstract both the Button and the Box into components (i.e. `TabButton` and `TabUnderline`, probably components in the same file is ok